generate.sh

```diff
@@ -88,6 +88,14 @@ ENVVARS="$(python ${BASE}/modules/scripts/config.py "${USERNAME}" "${REPONAME}")
 echo -e "\n${ENVVARS}\n" >> ${LOGFILE}
 eval $ENVVARS
 
+if [ -n "$TARGET_BRANCH" ]; then
+  git checkout "$TARGET_BRANCH" >> ${LOGFILE}
```

Comment:
.yaydoc.yml will only exist in the branch from which docs are generated. Did reading the branch from it ever made sense?